Tasks/AzureAppConfigurationImportV10/task.json

```diff
@@ -2,14 +2,14 @@
     "id": "fd186364-adb5-43c0-91ea-c5f9daa5b747",
     "name": "AzureAppConfigurationImport",
     "friendlyName": "Azure App Configuration Import",
-    "description": "Push key-values to an Azure App Configuration instance",
-    "helpUrl": "https://learn.microsoft.com/azure/azure-app-configuration/push-kv-devops-pipeline",
+    "description": "Import key-values to an Azure App Configuration instance",
+    "helpUrl": "https://learn.microsoft.com/azure/azure-app-configuration/azure-pipeline-import-task",
     "helpMarkDown": "Email AzureAppConfig@microsoft.com for questions.",
     "category": "Utility",
     "author": "Microsoft Corporation",
     "version": {
         "Major": 10,
-        "Minor": 247,
+        "Minor": 249,
```

Comment:
Just curious, how do we decide the minor version?

Reply:
Minor version is based on the current sprint version please see-> https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md